### PR TITLE
Adjust coordinator logs and add remove metadata-hash from compiled ocr2vrf contracts

### PR DESCRIPTION
**Pull request by vreff**
Demote coordinator bulk logs to trace, add debug logs for function execution time, and prevent solc from adding the metadata hash to ocr2vrf contract files.

### CODE REVIEW

File: contracts/scripts/native_solc8_15_compile_ocr2vrf [hunk 1]
```diff
@@ -9,7 +9,7 @@
 ROOT="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; cd ../../ && pwd -P )"
 
 solc-select use 0.8.15
-solc @openzeppelin/=$ROOT/node_modules/@openzeppelin/ --overwrite --optimize --optimize-runs 1000 \
+solc @openzeppelin/=$ROOT/node_modules/@openzeppelin/ --overwrite --optimize --optimize-runs 1000 --metadata-hash none \
```
Comment (makramkd) [inline contracts/scripts/native_solc8_15_compile_ocr2vrf:12]: Do you have to re-gen the gethwrappers? Or does this just affect the abi output?

Comment (vreff) [reply contracts/scripts/native_solc8_15_compile_ocr2vrf:12]: I regenned them, yeah it does.

Comment (vreff) [reply contracts/scripts/native_solc8_15_compile_ocr2vrf:12]: I regenned them, yeah it does.

**Comment by vreff** [comment]
Regenerating the go wrappers messed up some tests, not sure why but likely the recent changes to the contracts are interfering somehow. I'm leaving the metadata hash change out of this PR and will revisit it when I merge in the LoadTestConsumer contract change.

**Review by makramkd — APPROVED** [review]

**Pull request merged**